src/diffusers/models/unets/unet_3d_condition.py

```diff
@@ -503,6 +504,44 @@ def disable_freeu(self):
                 if hasattr(upsample_block, k) or getattr(upsample_block, k, None) is not None:
                     setattr(upsample_block, k, None)
 
+    # Copied from diffusers.models.unets.unet_2d_condition.UNet2DConditionModel.fuse_qkv_projections
+    def fuse_qkv_projections(self):
```

Comment:
fuse_qkv_projections support for UNet3DConditionModel was added after the last rebase commit